common/backintime.py

```diff
@@ -464,6 +464,36 @@ def createParsers(app_name='backintime'):
     snapshotsPathCP.set_defaults(func = snapshotsPath)
     parsers[command] = snapshotsPathCP
 
+    command = 'status'
+    nargs = 0
+    aliases.append((command, nargs))
+    description = 'Show a summary of the last snapshot from each profile.'
+    snapshotStatusCP =       subparsers.add_parser(command,
+                                                 epilog = epilogCommon,
+                                                 help = description,
+                                                 description = description)
```

Comment:
Please do not copy the fancy coding style of previous developers. ;) Just stick to PEP8 please.

```suggestion
    snapshotStatusCP = subparsers.add_parser(command,
                                             epilog=epilogCommon,
                                             help=description,
                                             description=description)
```